include/Gaffer/TweakPlug.h

```diff
@@ -123,6 +123,15 @@ class GAFFER_API TweakPlug : public Gaffer::ValuePlug
 			MissingMode missingMode = MissingMode::Error
 		) const;
 
+		template< typename T >
+		static void applyNumericTweakValue(
```

Comment:
Maybe just `applyNumericTweak()`, and return the result rather than pass `dest` by reference?

Reply:
I've done this, though I didn't like using the same name for this and the other function named applyNumericTweak. I realized the other function is private though, so I've just renamed it applyNumericDataTweak ... is that reasonable?